packages/manager/modules/at-internet-configuration/src/index.js

```diff
@@ -77,18 +77,19 @@ angular
         ...atInternetConfiguration.getConfig(Environment.getRegion()),
         ...(referrerSite ? { referrerSite } : {}),
       };
-      $http
-        .get('/me')
-        .then(({ data: me }) => me)
-        .catch(() => {})
-        .then((me) => {
-          atInternet.setDefaults({
+
+      atInternet.setDefaultsPromise(
+        $http
+          .get('/me')
+          .then(({ data: me }) => me)
+          .catch(() => {})
```

Comment:
won't it fail if the catch is before ? it'll return an empty object then me.country will break in the next "then" block ?

Reply:
If the code fails, the catch returns an empty object, and it won't break, only return `undefined` values.

```js
const me = {};

console.log({
  countryCode: me.country,
  currencyCode: me.currency && me.currency.code,
  visitorId: me.customerCode,
});
```

```js
{countryCode: undefined, currencyCode: undefined, visitorId: undefined}
```